CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -469,13 +470,18 @@ class CameraFragment : Fragment() {
          * should not store external references to this image, as these references will become
          * invalid.
          *
-         * @param image image being analyzed VERY IMPORTANT: do not close the image, it will be
-         * automatically closed after this method returns
+         * @param image image being analyzed VERY IMPORTANT: Analyzer method implementation must
+         * call image.close() on received images when finished using them. Otherwise, new images
+         * may not be received or the camera may stall, depending on back pressure setting.
+         *
          * @return the image analysis result
```

Comment:
This is obsolete, the image analysis no longer returns anything. Please remove.

Reply:
For some reason GitHub won't mark the lines if they are not part of your change. I'm referring to line 477 (the @return).